layers/+distributions/spacemacs-base/keybindings.el

```diff
@@ -235,6 +235,7 @@
   "cd" 'spacemacs/close-compilation-window)
 (with-eval-after-load 'compile
   (define-key compilation-mode-map "r" 'recompile)
+  (define-key evil-motion-state-map "gf" 'find-file-at-point)
```

Comment:
I still could not understand the difference between both if any, if I remember correctly both did the same. 

Thanks for your help 👍